codegen/codegen.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 const (
-	defaultURL = "https://releases.rancher.com/kontainer-driver-metadata/dev-v2.6/data.json"
+	defaultURL = "https://raw.githubusercontent.com/rancher/kontainer-driver-metadata/59bf03888f4f38d5aef44756802bf9830b049d61/data/data.json"
```

Comment:
Do we want to move the default away from the release site?

Reply:
Nope, thanks for calling it out - I used my dev commit for checking if tests pass and forgot updating it.

Reply:
I'll update it back to dev-v2.6 after I merge KDM changes